### DISCUSSION TITLE
how can I add colorpicker on the default rich text editor to change font colors?

### DISCUSSION THREAD

**Discussion by ngdevc** [GrapesJS/grapesjs#3843] [Q&A]
I'm going to change the font color using a color picker and I added it like this. but it's not working now. I think It's appeared due to 'rte.exec()' please help me how to change the font color. Thank you.

**Comment by ngdevc** [comment]
![image](https://user-images.githubusercontent.com/65579678/135786651-e6637e02-e56a-4723-b586-9fedc3523b0e.png)

This is my current code. Please help me how can i add the color picker.

**Comment by ngdevc** [comment]
i have solved it by change `event:"change"` to `event:"input"`.

**Comment by ahmadlimbada** [comment]
@artf @tortuc I was able to do it. However it's default HTML color picker. Is there any way to add spectrum color picker(same as one in style manager) in built-in RTE?
